### DISCUSSION TITLE
iOS: Cannot connect to server

### DISCUSSION THREAD

**Discussion by ypo777** [angristan/wireguard-install#235] [Help]
I have Wireguard VPN setup on Ubuntu 20.04 (LTS) x64 [DigitalOcean Server]. I used QR to add tunnel to my phone. 
But no internet when I check logs on my phone it fails to handshake with server. 

`" 2021-05-04 18:23:34.150
[NET] peer(sJ2L…VggQ) - Sending handshake initiation
2021-05-04 18:23:39.402
[NET] peer(sJ2L…VggQ) - Handshake did not complete after 5 seconds, retrying (try 19)"`

This is client conf: 
[Interface]
PrivateKey = Private
Address = 10.104.0.6/32,fd42:42:42::6/128
DNS = 94.140.14.14,94.140.15.15

[Peer]
PublicKey = Public
PresharedKey = PreshareKey
Endpoint = server_ip:57733
AllowedIPs = 0.0.0.0/0,::/0

This is server conf(wg0.conf):
`[Interface]
Address = 10.104.0.4/24/24,fd42:42:42::1/64
ListenPort = 57733
PrivateKey = Private
PostUp = iptables -A FORWARD -i eth0 -o wg0 -j ACCEPT; iptables -A FORWARD -i wg0 -j ACCEPT; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE; ip6tables -A FORWARD -i wg0 -j ACCEPT; ip6tables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
PostDown = iptables -D FORWARD -i eth0 -o wg0 -j ACCEPT; iptables -D FORWARD -i wg0 -j ACCEPT; iptables -t nat -D POSTROUTING -o eth0 -j MASQUERADE; ip6tables -D FORWARD -i wg0 -j ACCEPT; ip6tables -t nat -D POSTROUTING -o eth0 -j MASQUERADE

### Client test
[Peer]
PublicKey =Public
PresharedKey = PreShare
AllowedIPs = 10.104.0.6/32,fd42:42:42::6/128`

**Comment by x-FBi** [comment]
If he can connect to the server with ssh, chances are you're not being blocked.
I'd check with IPTables or maybe some other firewall.  netstat -nl
iptables -L
